test/system/scenarios/testMarketStopOrders.js

```diff
@@ -0,0 +1,18 @@
+const TestMarketStopOrders = (trader1, trader2, trader3) => {
+  console.log('******************** Scenario 4 Test market stop orders  *********************');
+  return trader1.init_state(trader1.config)
+    .then(() => trader2.init_state(trader2.config))
+    .then(() => trader3.init_state(trader3.config))
+    .then(() => trader1.placeLimitOrderSpread('B', 0.21, 1, 3, 0.01))
+    .then(() => trader1.placeLimitOrderSpread('S', 0.25, 1, 3, 0.01))
```

Comment:
A 'spread' is typically made of both Buy and Sell orders. Can you modify placeLimitOrderSpread to do that and squash these two lines into one.